### PR TITLE
feat: image component

### PR DESCRIPTION
**Pull request by NandhiniKrishnamoorthy**
Please review the image component .
       Round images are not responsive images and responsive image has done separately
       Please have a look on the preview:  https://styles-10.netlify.app/

### CODE REVIEW

File: css/images.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿.round-image {
```
Comment (dev-enforced) [inline css/images.css:1]: This class has repeated properties as that of square-image so you can write like this
.round-image,.square-image{
//style rules common to both
}
.round-image{
//style-rules applicable only to round-image
}

File: index.html [hunk 2]
```diff
@@ -277,6 +277,20 @@ <h4 class="m-auto my-1 content-size">Cards with shadow</h4>
             </div>
         </section>
 
+        <!--------------Image-------------->
+        <h2 class="m-auto content-size mt-2">Images</h2>
+        <section class="m-auto mt-1 px-0p5 py-1 container-wrapper">
+            <h4 class="m-auto mb-1 content-size">Round Image</h4>
+            <div class="container-flex-row">
+                <img src="images/round_image.jpg" class="round-image" />
```
Comment (dev-enforced) [inline index.html]: Do provide an alt attribute to all the images mentioned for better accessibility

**Review by dev-enforced — CHANGES_REQUESTED** [review]
The code looks good to me . You can merge it after correcting the below mentioned changes. For Vs code related files you can use a .gitignore so that vs code related settings should not be pushed onto Github

**Review by JowelTisso — APPROVED** [review]
Looks good

File: css/images.css (hunk 1 above)
Comment (NandhiniKrishnamoorthy) [reply css/images.css:1]: removed the repeated style rules and reused the code

File: index.html (hunk 2 above)
Comment (NandhiniKrishnamoorthy) [reply index.html]: Added alt property

**Comment by NandhiniKrishnamoorthy** [comment]
> The code looks good to me . You can merge it after correcting the below mentioned changes. For Vs code related files you can use a .gitignore so that vs code related settings should not be pushed onto Github

**Review by dev-enforced — APPROVED** [review]
Looks good to me. You can merge it @NandhiniKrishnamoorthy

**Pull request merged**